Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -14,6 +14,9 @@ public class MixedRealityToolkitInspector : UnityEditor.Editor
         private int currentPickerWindow = -1;
         private bool checkChange = false;
```

Comment:
>checkChange [](start = 21, length = 11)

I think that this is no longer used now that it's been deleted in the profile stuff down below - note that it's still SET in OnEnable but not referenced after that

Reply:
The checkChange or picker window? They both are still used I believe

Reply:
checkChange specifically - I only see it getting updated in OnEnable (but it's never used in an if statement anymore)

Reply:
Ahhh checkChange, I was looking at changed. Yes I’ll update